skrifa/src/string.rs

```diff
@@ -597,12 +597,72 @@ const LANGUAGE_ID_TO_BCP47: &[(u16, &str)] = &[
     (0x540a, "es-US"),        //Spanish
 ];
 
+fn _get_best_name(font: &FontRef, name_ids: Vec<StringId>) -> Option<String> {
```

Comment:
you can omit the leading `_`, pub/private is enforced by the compiler.

Reply:
Thanks for the info. I changed it.